include/z64.h

```diff
@@ -737,8 +737,8 @@ typedef struct {
     /* 0x242 */ s16 heartsEnvR[2];
     /* 0x246 */ s16 heartsEnvG[2];
     /* 0x24A */ s16 heartsEnvB[2];
-    /* 0x24E */ s16 unk_24E;
-    /* 0x250 */ s16 unk_250;
+    /* 0x24E */ s16 health;
```

Comment:
Do you happen to know the difference between `interfaceCtx.health` and `gSaveContext.health`? Just curious. If not. no worries, it'll be worked out in later documentation

Reply:
no, from what I gathered ( either in the codebase and through testing on emulator) it's never really used, same goes for the func that interact with it in z_lifemeter (Increase, Decrease and Save). in game it seem to only reflect the hp you had when entering the zone. it isn't documented in oot either, my guess is it's some unused/debug features that wasn't removed